Documentation/Storage-Configuration/Monitoring/ceph-monitoring.md

```diff
@@ -63,6 +63,32 @@ Ensure that the Prometheus server pod gets created and advances to the `Running`
 kubectl -n rook-ceph get pod prometheus-rook-prometheus-0
 ```
 
+At this point, the user can configure Prometheus end-point so dashboard can get metrics from Prometheus.
+Rook provides two parameters for this: prometheusEndpoint and prometheusEndpointSSLVerify. The first
+one is to specify the url of Prometheus instance meanwhile the second one controls whether SSL should be
+verified in case the Prometheus server is using HTTPS. The Following command can be used to get Prometheus url:
+
+```console
+echo "http://$(kubectl -n rook-ceph -o jsonpath={.status.hostIP} get pod prometheus-rook-prometheus-0):30900"
+```
+
+Following is an example of the changes to configure Prometheus end point (e.g., `cluster.yaml`).
+
+    ```YAML
+    apiVersion: ceph.rook.io/v1
+    kind: CephCluster
+    metadata:
+      name: rook-ceph
+      namespace: rook-ceph
+    [...]
+    spec:
+    [...]
+      dashboard:
```

Comment:
I'd suggest to remove all the context (remove lines 78-85) and just show the brief snippet from the cephcluster CR. The user should already be familiar with the cephcluster CR from other docs.
```suggestion
   spec:
      dashboard:
```